--- origsrc/xmore-1.0.1/xmore.c	2005-07-05 17:51:11.000000000 -0500
+++ src/xmore-1.0.1/xmore.c	2008-11-04 23:47:36.374389300 -0600
@@ -302,7 +302,11 @@
 #endif
   
   n = 0;
+#ifdef USE_XPRINT
   XtSetArg(args[n], XtNfromHoriz,       printbutton);            n++;
+#else
+  XtSetArg(args[n], XtNfromHoriz,       NULL);                   n++;
+#endif
   XtSetArg(args[n], XtNfromVert,        text);                   n++;
   XtSetArg(args[n], XtNlabel,           "Quit");      n++;
   quitbutton = XtCreateManagedWidget("quit", commandWidgetClass, form, args, n);
